DataFormats/Provenance/src/BranchType.cc

```diff
@@ -78,11 +78,13 @@ namespace edm {
     std::string const fileIndex = "FileIndex";
     std::string const indexIntoFile = "IndexIntoFile";
     std::string const mergeableRunProductMetadata = "MergeableRunProductMetadata";
+    std::string const processBlockHelper = "ProcessBlockHelper";
     std::string const eventHistory = "EventHistory";
     std::string const eventBranchMapper = "EventBranchMapper";
 
     std::string const eventSelections = "EventSelections";
     std::string const branchListIndexes = "BranchListIndexes";
+    std::string const eventToProcessBlockIndexes = "EventToProcessBlockIndexes";
```

Comment:
Thought for the future. Given we now have `std::string_view` which can be `constexpr` we should probably change all of these and the accessor methods to use `std::string_view` just to avoid the runtime intialization.

Reply:
I see that for C++20 std::string constructor is also `constexpr` capable so maybe the idea is moot.